Formula/ooniprobe.rb

```diff
@@ -169,10 +167,6 @@ def install
     (var/"lib/ooni").mkpath
   end
 
-  def post_install
-    system bin/"ooniresources", "--update-inputs", "--update-geoip"
```

Comment:
Yes, it's now run as part of `setup.py` (https://github.com/TheTorProject/ooni-probe/blob/master/setup.py#L166)